### PR TITLE
UI improvements on subscribed applications

### PR DESCRIPTION
**Pull request by sahare**
https://github.com/open-cluster-management/backlog/issues/10745

Adding git tag and commit hash for Git channels
Adding reconcile rate for Git and Helm channel types

Screen Shot 2021-04-09 at 3 48 44 PM

### CODE REVIEW

**Comment by sahare** [comment]
@swopebe Could you please review the texts that are added to nls/platform.properties file and let me know if they need any modifications? Thanks!

**Review by chenz4027 — APPROVED** [review]
lgtm!

File: nls/platform.properties [hunk 1]
```diff
@@ -589,6 +595,8 @@ tooltip.creation.app.github.accessid = The access token if this is a private Git
 tooltip.creation.app.github.branch = The branch of the Git repository.
 tooltip.creation.app.github.path = The location of the resources on the Git repository.
 tooltip.creation.app.github.reconcileOption = With the Merge option, new fields are added and existing fields are updated in the resource. Choose to merge if resources are updated after the initial deployment. If you choose to replace, the existing resource is replaced with the Git source.
+tooltip.creation.app.github.desiredCommit = If you want to subscribe to a specific commit, you need to specify the desired commit hash. You may need to specify git-clone-depth annotation if your desired commit is older than the last 20 commits.
```
Comment (swopebe) [inline nls/platform.properties:598]: Please use "might" instead of "may."

File: nls/platform.properties [hunk 2]
```diff
@@ -589,6 +595,8 @@ tooltip.creation.app.github.accessid = The access token if this is a private Git
...
+tooltip.creation.app.github.tag = If you want to subscribe to a specific tag, you need to specify the tag. If both Git desired commit and tag annotations are specified, the tag will be ignored. You may need to specify git-clone-depth annotation if your desired commit of the tag is older than the last 20 commits.
```
Comment (swopebe) [inline nls/platform.properties:599]: Change: will be ignored 
to 

is ignored

File: nls/platform.properties [hunk 3]
```diff
@@ -608,6 +616,8 @@ tooltip.creation.app.settings.existingRule=If available in the application names
 tooltip.creation.app.settings.localClusters=Deploy application resources on local cluster only
 tooltip.creation.app.settings.onlineClusters=Deploy your application resources on all online clusters, including your local cluster.
 tooltip.creation.app.settings.onlineClustersOnly=Deploy your application resources on all online clusters.
+tooltip.creation.app.reconcileRate = The frequeny of resource reconciliation that will be used as a global repository setting. The default setting is medium that checks for changes to apply every 3 minutes and re-applies all resources ever 15 minutes even if there is no change. Select low to reconcile every hour and high to reconcile every 2 minutes. By selecting off, the deployed resources are not automatically reconciled.
```
Comment (swopebe) [inline nls/platform.properties:619]: frequency misspelled here

Comment (swopebe) [reply nls/platform.properties:619]: change: that will be used
to 
that is used

Comment (swopebe) [reply nls/platform.properties:619]: The `medium` default setting checks for changes to apply every three minutes and re-applies all resources every 15 minutes, even without a change. Select `low` to reconcile every hour.  Select `high` to reconcile every two minutes. If you select `off`, the deployed resources are not automatically reconciled.

Many changes needed -- a bit confusing on the first line (subject/predicate changes), other small fixes.

**Comment by swopebe** [comment]
@sahare it looks like it merged auto before I could review... can you see the changes I added in comments?

**Pull request merged**